Nos/Views/Profile/ProfileHeader.swift

```diff
@@ -152,7 +152,7 @@ struct ProfileHeader: View {
             NavigationView {
                 BioSheet(author: author)
                     .background(Color.bioBgGradientBottom)
-                    .nosNavigationBar(title: .localizable.profile)
+                    .navigationBarTitle(String(localized: .localizable.profileTitle), displayMode: .inline)
```

Comment:
@mplorentz @martindsq Rethinking this since I missed that we need the Clarity font. So `.nosNavigationBar(title:)` has all the good stuff -- the Clarity font at the proper size and weight, the foreground color -- but it also has this:

`.padding(.leading, 14)`

That's why this wasn't centered, but it seems the padding is necessary for all nav bars that appear from the slide-over menu. I think. I wouldn't be unhappy if that slide-over menu design changed to something else, but for now we need to find a good solution. Thoughts? Do we just keep two navigationBar modifiers: one with leading padding and the other without?